config/template.go

```diff
@@ -118,6 +118,11 @@ type TemplateConfig struct {
 	// and causes an error if a relative path tries to traverse outside that
 	// prefix.
 	SandboxPath *string `mapstructure:"sandbox_path"`
+
+	// MapToEnvironmentVariable is the name of the environment variable this
+	// template should map to. This field is only applicable used when
```

Comment:
"[..] the environment variable this template should map to."

I'm not sure what you mean here by "map to". Would you mind clarifying this bit some so it is more obvious?

Reply:
Maybe take inspiration from the PR message? It makes more sense to me.

Reply:
Perhaps a longer comment is warranted here. When `MapToEnvironmentVariable` is specified, Vault Agent will interpret this template as an environment variable template and output its contents to an environment variable rather than a file. Essentially this field is mutually exclusive with `Destination *string`.

Reply:
Yes please. You explained it perfectly here... if you could distill that into a comment I (and other future readers) would greatly appreciate it. Thanks!